mllib/src/main/scala/org/apache/spark/ml/classification/LinearSVC.scala

```diff
@@ -47,7 +47,7 @@ private[classification] trait LinearSVCParams extends ClassifierParams with HasR
 /**
  * :: Experimental ::
  *
- * Linear SVM Classifier (https://en.wikipedia.org/wiki/Support_vector_machine#Linear_SVM)
+ * [[https://en.wikipedia.org/wiki/Support_vector_machine#Linear_SVM  Linear SVM Classifier]]
```

Comment:
I checked several other scala files like, GBTRegressor.scala, DecisonTreeRegressor.scala etc. URLs are in the format of <a href>...</a>.